pkg/scheduler/scheduler_test.go

```diff
@@ -5245,6 +5245,71 @@ func TestScheduleForTASPreemption(t *testing.T) {
 				},
 			},
 		},
+		"With pods count usage pressure on nodes: only low priority workload is preempted": {
+			// This test case demonstrates the baseline scenario where there
+			// is only one low-priority workload and it gets preempted even if node has pods count usage pressure.
+			nodes: []corev1.Node{
+				*singleNode.Clone().
```

Comment:
I checked locally and it seems this to be the root cause for the failures of other tests, probably the Clone does not work as expected, maybe revert the wrappers changes and just create the locally for this test

Reply:
Thank you for checking that. I found my poor mistake in another place :)